code/index.html

```diff
@@ -1,22 +1,136 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>The Fashionable Update - News & inspiration from the design world</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- copy paste meta tag if not working and perhaps add Favicon -->
+
+</head>
+
+<body>
+    <!-- Logotype, Name of site and Menu -->
+    <header class="header">
+        <img class="logo" src="./Silhuette-Logotype-with-circle.svg" alt="Logotype">
+        <h1 class="newspaper-name">The Fashionable Update</h1>
+        <!-- <input id="menu" type="checkbox">
+        <label for="menu"></label> -->
+
+        <nav class="navigation-menu">
+            <div class="navigation-item">HOME</div>
+            <div class="navigation-item">ABOUT</div>
+            <div class="navigation-item">CONTACT</div>
+        </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+
+    <!-- The Big News -->
+    <section class="container-big-news">
+        <div class="main-news-card">
+            <div class="couture">
+                <img src="./Couture.jpg" alt="Augusta Lundin Exhibition">
+                <div class="heading-couture">
```

Comment:
Hmm, this `div` contains both an `h2` and a `p`. What could be a more descriptive class name for this?